doc-src/examples/s3/client/get_object/03_streaming_data_to_a_block.rb

```diff
@@ -1,6 +1,8 @@
 # WARNING: yielding data to a block disables retries of networking errors
+# However truncation of the body will be retried automatically using a range request
 File.open('/path/to/file', 'wb') do |file|
-  s3.get_object(bucket: 'bucket-name', key: 'object-key') do |chunk|
+  s3.get_object(bucket: 'bucket-name', key: 'object-key') do |chunk, headers|
+    # headers['content-length']
```

Comment:
Was this intended to be commented in the example?

Reply:
Yeah - intended to document the parameter, but its not actually used here.